src/Inkluzitron/Modules/UserModule.cs

```diff
@@ -39,68 +41,121 @@ protected override void AfterExecute(CommandInfo command)
-                return;
-            }
-
-            var gender = userDb.Gender == Gender.Unspecified ?
-                "nemá preferované oslovení." :
-                $"je {userDb.Gender.GetDisplayName()}.";
-
-            await ReplyAsync(string.Format(genderMsg, Format.Sanitize(await UsersService.GetDisplayNameAsync(user)), gender));
-        }
-
-        [Command("pronouns set he")]
-        [Alias("pronouns set him", "pronouns set on", "osloveni set he", "osloveni set him", "osloveni set on")]
-        [Summary("Nastaví svoje preferované oslovení jako mužské - he/him, on.")]
-        public Task SetGenderMaleAsync()
-            => SetGenderAsync(Gender.Male);
-
-        [Command("pronouns set she")]
-        [Alias("pronouns set her", "pronouns set ona", "osloveni set she", "osloveni set her", "osloveni set ona")]
-        [Summary("Nastaví svoje preferované oslovení jako ženské - she/her, ona.")]
-        public Task SetGenderFemaleAsync()
-            => SetGenderAsync(Gender.Female);
-
-        [Command("pronouns unset")]
-        [Alias("pronouns set other", "osloveni unset", "osloveni set other")]
-        [Summary("Nastaví neutrální oslovení.")]
-        public Task UnsetGenderAsync()
-            => SetGenderAsync(Gender.Unspecified);
-
-        public async Task SetGenderAsync(Gender gender)
-        {
+            }
+
+            var userDb = await DbContext.GetUserEntityAsync(user);
+            if (userDb == null)
+            {
+                await ReplyAsync(string.Format(notFoundMsg, Format.Sanitize(await UsersService.GetDisplayNameAsync(user))));
+                return;
+            }
+
+            var gender = userDb.Gender == Gender.Unspecified ?
+                "nemá preferované oslovení." :
+                $"je {userDb.Gender.GetDisplayName()}.";
+
+            await ReplyAsync(string.Format(genderMsg, Format.Sanitize(await UsersService.GetDisplayNameAsync(user)), gender));
+        }
+
+        [Command("pronouns set he")]
+        [Alias("pronouns set him", "pronouns set on", "osloveni set he", "osloveni set him", "osloveni set on")]
+        [Summary("Nastaví svoje preferované oslovení jako mužské - he/him, on.")]
+        public Task SetGenderMaleAsync()
+            => SetGenderAsync(Gender.Male);
+
+        [Command("pronouns set she")]
+        [Alias("pronouns set her", "pronouns set ona", "osloveni set she", "osloveni set her", "osloveni set ona")]
+        [Summary("Nastaví svoje preferované oslovení jako ženské - she/her, ona.")]
+        public Task SetGenderFemaleAsync()
+            => SetGenderAsync(Gender.Female);
+
+        [Command("pronouns unset")]
+        [Alias("pronouns set other", "osloveni unset", "osloveni set other")]
+        [Summary("Nastaví neutrální oslovení.")]
+        public Task UnsetGenderAsync()
+            => SetGenderAsync(Gender.Unspecified);
+
+        [Command("duck set")]
```

Comment:
```suggestion
        [Command("duck set")]
        [Alias("kachna set")]
```
Aby to bylo konzistentní s kachna modulem.